apps/src/templates/projects/projectsRedux.js

```diff
@@ -385,6 +395,21 @@ const updateProjectNameOnServer = (project) => {
   };
 };
 
+const remixProjectOnServer = (project) => {
+  return (dispatch) => {
+    $.ajax({
+      url: `projects/${project.projectType}/${project.id}/remix`,
+      method: 'GET',
+      type: 'json',
+      contentType: 'application/json;charset=UTF-8',
+      data: JSON.stringify(project)
+    }).done((data) => {
```

Comment:
once you get your request to stop following the redirect, parse and return the new channel id (or complete path to the project) from the response here